src/games/strategy/triplea/ui/ExtendedStats.java

```diff
@@ -68,11 +68,9 @@ private void fillExtendedStats(final GameData data) {
         m_statsExtended = statsExtended.toArray(new IStat[statsExtended.size()]);
       }
       // add number of techs
-      if (true) {
-        final List<IStat> statsExtended = new ArrayList<>(Arrays.asList(m_statsExtended));
-        statsExtended.add(new TechCountStat());
-        m_statsExtended = statsExtended.toArray(new IStat[statsExtended.size()]);
-      }
+      final List<IStat> statsExtended = new ArrayList<>(Arrays.asList(m_statsExtended));
```

Comment:
Weird for whatever reson the compiler thinks, that this variable is already declared...
It is, but only in enclosed if statements...